### ISSUE TITLE
iptsd installed but not enabled in systemd after first install SB2

### ISSUE THREAD

**Issue by grandrew** [linux-surface/linux-surface#271]
I recently installed ubuntu using instructions provided on main github page. Everything went smooth except that iptsd was not enabled so I was confused why touchscreen does not work.

### Environment

- Hardware model: Surface Book 2
- Kernel version: 5.8.1-surface
- Distribution: Ubuntu

Something might be missing in iptsd package

**Comment by qzed** [comment]
I'm not sure how things are handled on Ubuntu (or Debian, or Fedora) with regards to auto-enabling stuff (I'd love to hear what the official stance on that is), but on Arch it's generally frowned upon. Our current packages are all built with the intent that you'll have to manually enable services etc. The only notable exception to this is the `linux-surface-secureboot-mok` package, which has some post-install automation.

**Comment by StollD** [comment]
Fedora doesn't enable systemd services by default. Debian usually does, but given this issue, I think it is a policy instead of an automated detection.

Given that the installation guide on the wiki already tells you to run `systemctl enable iptsd`, I don't think it is a huge issue to be honest. And it keeps the process similar on all supported distributions.

**Comment by qzed** [comment]
> given this issue, I think it is a policy instead of an automated detection.

Yeah, I think that would be handled by the post-install script of the package.

> And it keeps the process similar on all supported distributions.

Hmm yeah, depends what we want to strive for: Consistency across our packages or adhering to the respective distribution standards. I'd kinda be in favor of the latter. I've looked a bit around and found this: https://unix.stackexchange.com/questions/274624/automatically-enable-systemd-services-installed-using-deb (see first answer, requires `dh-systemd` package when building, if there's a `postinst` script, that requires a `#DEBHELPER#` line, also requires some additions to the systemd service as pointed out in the third answer).
